assets/js/components/notifications/BannerNotification/index.js

```diff
@@ -197,32 +206,16 @@ function BannerNotification( {
 	const closedClass = isClosed ? 'is-closed' : 'is-open';
 	const inlineLayout = 'large' === format && 'win-stats-increase' === type;
 
-	let layoutCellProps;
-	if ( 'large' === format ) {
-		layoutCellProps = inlineLayout
-			? {
-					mdSize: 5,
-					lgSize: 8,
-					smOrder: 2,
-					mdOrder: 1,
-			  }
-			: {
-					mdSize: 6,
-					lgSize: 8,
-					smOrder: 2,
-					mdOrder: 1,
-			  };
-	} else if ( 'small' === format ) {
-		layoutCellProps = {
-			smSize: 3,
-			mdSize: 7,
-			lgSize: 11,
-		};
-	} else {
-		layoutCellProps = {
-			size: 12,
-		};
-	}
+	const imageCellSizeProperties = getImageCellSizeProperties( format );
+	const imageCellOrderProperties = getImageCellOrderProperties( format );
+	const contentCellOrderProperties = getContentCellOrderProperties( format );
+	const contentCellSizeProperties = getContentCellSizeProperties( {
+		format: imageCellSizeProperties,
```

Comment:
Looks like an object was being passed as `format` before the refactoring I did. https://github.com/google/site-kit-wp/commit/e4e839d2a97b1ac03b59a1e76639331879a34d61#diff-52e8414857d98d3b55ae8f581b6012c13fa2aa60f8f4010e39ac27c2e74fab48R213

Should this simply be `format` ? 
```suggestion
		format,
```

Reply:
Correct @aaemnnosttv Not sure what happened here 🤦  Updated.